### ISSUE TITLE
Show activities that are not bookable (YET!)

### ISSUE THREAD

**Issue by rayedgar** [bookingactivities/booking-activities#146]
Is there a way to show upcoming activities that will show on the calendar that are bookable in say 21 days? (hide/display option)
If I set the availability now before the threshold of 'at the earliest' they activities are NOT displayed.

We do want to inform the users that the activity is  coming up, but due to admin reasons we cannot let them book yet.

**Comment by yoancutillas** [comment]
No, the 'at the earliest' option cuts the events interval at the source: future events are not retrieved at all.

However, you can display a global calendar displaying all the events for advisory purposes only, and another booking form to book the events.
- Go to Booking Activities > Booking forms > create new 
- Delete all fields except the "Calendar" field
- Go to Calendar settings > Filters tab > select the desired calendars and activities
- You can configure it to redirect the customer to the booking form page when he / she clicks the event in the Actions tab ([docs](https://booking-activities.fr/en/docs/user-documentation/advanced-use-of-booking-activities/add-product-to-cart-or-redirect-to-url-from-calendar/))
- Copy / paste the shortcode on the desired page

**Comment by rayedgar** [comment]
Thanks for your reply.. Im afraid that would not be workable and might be even more confusing to click an activity through another calendar where you cant book the activity (yet).  Is there a custom way to override the 'at the earliest' option so that it would retrieve the activities and display them? but not be bookable.  I see the logic here of course. But It would make the booking so more flexible. I think it would be a future request to be able to see upcoming events that are not bookable yet.

**Comment by yoancutillas** [comment]
Sure, you can do that with custom code.
If it can help, here is an example of custom code that prevent the booking if certain conditions are not met: #74, you can adapt this code with your own conditions.

I note your feature request, thank you!

**Comment by rayedgar** [comment]
I will look into it. many thanks
